config_converter/config_mapper/config_mapper.py

```diff
@@ -30,15 +30,51 @@
 _SUPPORTED_PLUGINS = ['in_tail']
 
 
+def initialize_stats(d: config_pb2.Directive) -> dict:
+    """Initializes the stats dict to print out."""
+    stats = {
+        'attributes_num': get_num_attrs(d),
+        'attributes_recognized': 0,
+        'attributes_unrecognized': 0,
+        'attributes_skipped': 0,
+        'entities_num': get_num_ents(d),
+        'entities_skipped': 0,
+        'entities_unrecognized': 0,
+        'entities_recognized_success': 0,
+        'entities_recognized_partial': 0,
+        'entities_recognized_failure': 0
+    }
+    return stats
+
+
+def get_num_attrs(d: config_pb2.Directive) -> int:
```

Comment:
1. This seemed an internal function and no external caller would use this function, so make the naming as a private function suggested by style guide: https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations.

2. When I see the function name, I expect it returns the # of attributes of this level of directives, but this function returns the # of attributes of this directive + # of attribute from the children directives.

I suggest `get_aggregated_num_attributes`, or name it something else we know that it aggregates attributes in children directives. What do you think? CC @davidbtucker

Reply:
I think these are both good suggestions. I think the name is very long though - do you have any suggestions about shorter ones that can still convey the point?